tests/unit_tests/test_wave_06.py

```diff
@@ -95,10 +95,10 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    assert item_f in tai.inventory
-    assert item_c in jesse.inventory
-    assert item_c not in tai.inventory
-    assert item_f not in jesse.inventory
+    assert [item_a, item_b, item_f] == tai.inventory
+    assert [item_d, item_e, item_c] == jesse.inventory
```

Comment:
This is a thorough check for all of the inventory items.

Since this test results in a swap occurring, but we don't have a clear specification on where items should end up when swapped, a length check with an exhaustive set of `in` checks would be the most flexible way to check that items are where they should be.

Checking against a literal list requires that the order be identical to compare equal, but we shouldn't depend on a particular order after a swap occurs.